index.php

```diff
@@ -1,3 +1,8 @@
 <?php get_header(); ?>
   <h1>This is my index.</h1>
 <?php get_footer(); ?>
+
+<div class='container'>

```

Comment:
Move this container above the footer, otherwise it will appear below the footer.